hack/local-up-cluster.sh

```diff
@@ -90,6 +90,9 @@ PRESERVE_ETCD="${PRESERVE_ETCD:-false}"
 # enable kubernetes dashboard
 ENABLE_CLUSTER_DASHBOARD=${KUBE_ENABLE_CLUSTER_DASHBOARD:-false}
 
+# enable Kubernetes-CSI snapshotter
+ENABLE_CSI_SNAPSHOTTER=${ENABLE_CSI_SNAPSHOTTER:-true}
```

Comment:
@pohly do you mind switching this to off as default?

Reply:
Because most users of the script won't need it?

That's okay for me, I can always just enable it via the command line snippet that I use for invoking the script. I am less sure about users who need it but don't know about it yet...

Reply:
Changed, force-pushed.

Reply:
I'm fine with this.